pkg/project/admission/lifecycle/admission.go

```diff
@@ -60,6 +60,11 @@ func (e *lifecycle) Admit(a admission.Attributes) (err error) {
 	if len(a.GetNamespace()) == 0 {
 		return nil
 	}
+	// always allow a SAR request through, the SAR will return information about
+	// the ability to take action on the object, no need to verify it here.
+	if isSubjectAccessReview(a) {
```

Comment:
wow, I completely missed that.  Does it make sense to move that entire check up to this line (sans the terminating piece which can stay).  Looks like it covers everything we want it can just take priority.